package-lock.json

```diff
@@ -111,6 +111,21 @@
         "webpack-merge": "^5.8.0"
       }
     },
+    "../../libraries/madie-models": {
+      "name": "@madie/madie-models",
+      "version": "1.2.8",
+      "license": "ISC",
```

Comment:
The local config got committed. I think the only way to get it to point at the npm version is to re-run `npm install`.

Reply:
I think i got this fixed